scenarios/triso/deployment_calcs/lwr_info.csv

```diff
@@ -1,121 +1,122 @@
+,Reactor name,State,Reactor type,Reactor vendora,Core size (number of assemblies),Startup date (year) b,License expiration (year),Actual retirement (year),power_cap
```

Comment:
What does the `b` do in `Startup date (year) b`? Also, consider adding units to `power_cap` like they are in core size, startup date, license expiration. Possibly also change it to match the others (Power cap).

Reply:
Good idea on the units!
```suggestion
,Reactor name,State,Reactor type,Reactor vendora,Core size (number of assemblies),Startup date (year) b,License expiration (year),Actual retirement (year),power_cap(MWe)
```

The b is an artifact of pulling from the website, I'm not sure why it's there.